pkg/recommendation/recommender/hpa/registry.go

```diff
@@ -70,7 +74,7 @@ func NewHPARecommender(recommender apis.Recommender, recommendationRule analysis
 	}
 
 	return &HPARecommender{
-		*replicasRecommender,
+		*(replicasRecommender.(*replicas.ReplicasRecommender)),
```

Comment:
What's this line means?

Reply:
NewReplicasRecommender return `recommender.Recommender` interface